pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -40,10 +46,16 @@ func (t *ofTable) Status() TableStatus {
 }
 
 func (t *ofTable) GetMissAction() MissActionType {
+	t.RLock()
+	defer t.RUnlock()
```

Comment:
ditto

Reply:
t.missAction is also a readable attribute, but not modifiable.

Reply:
Got it.